ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -174,6 +174,19 @@ <h3>Target details:</h3>
                             <mat-icon matTooltip="Download configured shards as JSON" matTooltipPosition="above">download</mat-icon>
                         </button></span>
                 </p>
+                <p class="point" *ngIf="(selectedMigrationType ==='dproc') && !(selectedMigrationMode ==='Schema')">
```

Comment:
better to compare the selected migration type with MigrationTypes.dataprocMigration to avoid errors and renaming at multiple places. Same for migration mode schema

Reply:
same comment for other places in this file

Reply:
As discussed this will need to add extra properties to component and will be a generic change (not directly related to dataproc integration)

Can be taken be as a seperate PR